openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
I understand. It is usually bad practice not to initialize variables at all and hope for raised errors. I think I had some problems with this not being initialized somewhere and added it. At least my IDE was complaining that this could fail, since is has never been initialized and does not even exist. Meaning it could fail, but for the wrong reasons. (I guess we are getting back to the old debate about duck-typing :). Anyway, I can remove it.